launchpad-frontend/src/myApplicationPage/myApplication.js

```diff
@@ -0,0 +1,334 @@
+        company: "Amazon",
+        duration: "New Grad",
+        location: "Vancouver, BC",
+        status: "Pending",
+        logo: "https://seeklogo.com/images/A/amazon-icon-logo-8F577E5C31-seeklogo.com.png",
+        date: "2023-10-21",
+        action: "Complete",
+        imageText:"Amazon Logo"
+      },
+      {
+        title: "Software Engineer, Junior",
+        company: "Microsoft",
+        duration: "New Grad",
+        location: "Toronto, ON",
+        status: "Pending",
+        logo: "https://upload.wikimedia.org/wikipedia/commons/thumb/4/44/Microsoft_logo.svg/2048px-Microsoft_logo.svg.png",
+        date: "2023-10-21",
+        action: "Complete",
+        imageText:"Logo",
+        redirect:"app_link"
+      },
+]
+
+// top banner
+function UserApplicationsInfo(){
+    return(
+        <Card sx={{ ...MyApplicationStyles.cardMargin, paddingTop: '15px', paddingBottom: '15px' }}>
+            <CardContent>
+                <div style={{ marginBottom: '20px', marginLeft:'30px' }}>
+                    <div style={{ display: 'flex', justifyContent: 'space-between', alignItems: 'center' }}>
+                        <div>
+                        <Typography variant="h6" component="div" style={{ fontSize: '18px', fontWeight: 'bold' }}>
+                            Application Submitted
+                        </Typography>
+                        </div>
+                        <div style={{ display: 'flex', alignItems: 'center' }}>
+                        <Typography variant="h6" component="div" style={{ fontSize: '17px', marginRight: '520px', color: '#C1C1C1', fontWeight:'bold' }}>
+                            60
+                        </Typography>
+                        </div>
+                    </div>
+                </div>
+
+                <div style={{ borderTop: '1px solid #ccc' }}></div>
+
+                <div style={{ marginTop: '20px', marginLeft:'30px' }}>
+                    <div style={{ display: 'flex', justifyContent: 'space-between', alignItems: 'center' }}>
+                        <div>
+                        <Typography variant="h6" component="div" style={{ fontSize: '18px', fontWeight: 'bold' }}>
+                            Selected for Interview
+                        </Typography>
+                        </div>
+                        <div style={{ display: 'flex', alignItems: 'center' }}>
+                        <Typography variant="h6" component="div" style={{ fontSize: '17px', marginRight: '520px', color: '#C1C1C1',fontWeight:'bold'  }}>
+                            5
+                        </Typography>
+                        </div>
+                    </div>
+                </div>
+            </CardContent>
+        </Card>
+    );
+}
+
+// bottom banner
+function UserApplications(){
+    const [savedExpanded, setSavedExpanded] = useState(false);
+
+    return(
+        <Card sx = {MyApplicationStyles.cardMargin}>
+            <TableContainer>
+                <Table>
```

Comment:
very tiny nit-pick: the rest of the pages have the 20px border-radius, if we could add that in here that would be great.